src/cpp/session/modules/SessionConnections.R

```diff
@@ -161,9 +161,18 @@ options(connectionObserver = list(
 })
 
 .rs.addJsonRpcHandler("get_new_connection_context", function() {
-  context <- list()
+   context <- list(
+      connectionsList = list(
+         list(
+            package = .rs.scalar("sparklyr"),
```

Comment:
No need to change this yet but I think we will eventually want `sourceName = 'package'` and `sourceType = 'sparklyr'` rather than `package = 'sparklyr'` as we'll soon have other sources of connections.